overlay/files-ubuntu-arm-rpi/etc/cos/bootargs.cfg

```diff
@@ -0,0 +1,18 @@
+set kernel=/boot/vmlinuz
+
+# Note on RPI bootargs
+# We additionally set modprobe.blacklist=vc4 as certain Displays are not supported by vc4.
+# As cOS main target is cloud and not graphics usage, we blacklist it to avoid 
```

Comment:
old reference to "cOS".  This should be Kairos now, yeah?

Reply:
i copied simple the layer of the other flavor, i thought i better leave it all like it is

Reply:
fixed